src/Sylius/Component/Attribute/AttributeType/OptionListAttributeType.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * @author    Fernando Caraballo Ortiz <fernando.ortiz@reiss.com>
+ * @date      28/10/2016
+ * @copyright Copyright (c) Reiss Clothing Ltd.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Attribute\AttributeType;
+
+use Sylius\Component\Attribute\Model\AttributeValueInterface;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+
+/**
+ * @author Fernando Caraballo Ortiz <fernando.ortiz@reiss.com>
+ */
+class OptionListAttributeType implements AttributeTypeInterface
+{
+    const TYPE = 'option_list';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getStorageType()
+    {
+        return AttributeValueInterface::STORAGE_ARRAY;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getType()
+    {
+        return static::TYPE;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function validate(AttributeValueInterface $attributeValue, ExecutionContextInterface $context, array $configuration)
+    {
```

Comment:
in theory you should implement this to validate that the values you have exist within the OptionList configuration... see `TextAttributeType`